chart/templates/NOTES.txt

```diff
@@ -20,18 +20,19 @@ under the License.
 
 Your release is named {{ .Release.Name }}.
 
+{{- $redis_is_deployed_by_chart := not (or .Values.redis.brokerURLSecretName .Values.redis.brokerURL) }}
```

Comment:
I think this is breaking current implementation, brokerURLSecretName was used in case redis was deployed as well. which is probably something we want to avoid

Reply:
Yeah but it's not actually used: https://github.com/apache/airflow/search?q=brokerURLSecretName
So now I suggest making use of it.